src/Datadog.Trace.DuckTyping/DuckTypeExceptions.cs

```diff
@@ -0,0 +1,172 @@
+using System;
+using System.Reflection;
+#pragma warning disable SA1649 // File name must match first type name
+#pragma warning disable SA1402 // File may only contain a single class
+
+namespace Datadog.Trace.DuckTyping
+{
+    /// <summary>
+    /// DuckType Exception
+    /// </summary>
+    public class DuckTypeException : Exception
+    {
+        internal DuckTypeException(string message)
+            : base(message)
+        {
+        }
+    }
+
+    /// <summary>
+    /// DuckType property can't be read
+    /// </summary>
+    public class DuckTypePropertyCantBeReadException : DuckTypeException
```

Comment:
Can you add test cases to so that each exception in this file has at least one test case?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/946/commits/e3020964dcbb2b80b94368777abbdd5b1de4e248.